langchain/vectorstores/qdrant.py

```diff
@@ -197,6 +197,7 @@ def similarity_search(
         query: str,
         k: int = 4,
         filter: Optional[MetadataFilter] = None,
+        score_threshold: Optional[float] = None,
```

Comment:
```suggestion
        *,
        score_threshold: Optional[float] = None,
```

Reply:
@eyurtsev this should be before the filter kwarg, no?